s/features/chat/api/cores/persistence/mock-chat-persistence.ts

```diff
@@ -101,7 +101,7 @@ export async function mockChatPersistence(storage: FlexStorage) {
 				await chatTables.mutes.create(
 					...userIds.map(userId => ({userId}))
 				)
-				events.mutes.publish({userIds})
+				userIds.map(i => this.isMuted(i) === true ? null : events.mutes.publish({userIds}))
```

Comment:
this doesn't quite seem right...

- we want to avoid creating mutes in the table when the mute already exists